pallets/ddc-customer-accounts/src/lib.rs

```diff
@@ -43,6 +43,7 @@ pub struct UnlockChunk<Balance: HasCompact> {
 pub struct Bucket<AccountId> {
 	bucket_id: u128,
 	owner_id: AccountId,
+	cluster_id: Option<u128>,
 	public_availability: bool,
```

Comment:
We should use the `ClusterId` type from the ddc-primitives that is `H160` now.

Reply:
Ok